src/java.base/share/classes/java/io/SimpleIO.java

```diff
@@ -0,0 +1,1037 @@
+     *
+     * @implNote The result represents the content split at line terminators.
+     */
+    public static List<String> readLines(Path path) {
+        Objects.requireNonNull(path, "filename must not be null");
+        try {
+            return Files.readAllLines(path, StandardCharsets.UTF_8);
+        } catch (IOException ex) {
+            throw new UncheckedIOException(ex);
+        }
+    }
+    /**
+     * The contents of the file at {@code path} are read as a string.
+     * For example:
+     * {@snippet lang="java":
+     * var text = """
+     *    "Hope" is the thing with feathers
+     *    That perches in the soul
+     *    And sings the tune without the words
+     *    And never stops - at all
+     *    """;
+     * write("data.txt", text); // @highlight substring="write"
+     * var data = read("data.txt");
+     * print(data);
+     * }
+     * will print on the output stream as:
+     * {@snippet lang="text":
+     * "Hope" is the thing with feathers
+     * That perches in the soul
+     * And sings the tune without the words
+     * And never stops - at all
+     *
+     * }
+     *
+     * @implSpec Line terminators are normalized the platform line separator.
+     *
+     * @param filename  file name string of file to be written
+     * @param content  string content of the file
+     * @throws UncheckedIOException wrapping an IOException if an io error occurs.
+     */
+    public static void write(String filename, String content) {
+        Objects.requireNonNull(filename, "filename must not be null");
+        Objects.requireNonNull(content, "content must not be null");
+        write(Path.of(filename), content);
+    }
+
+    /**
+     * The contents of the file at {@code path} are read as a string.
+     * For example:
+     * {@snippet lang="java":
+     * var text = """
+     *    "Hope" is the thing with feathers
+     *    That perches in the soul
+     *    And sings the tune without the words
+     *    And never stops - at all
+     *    """;
+     * var path = Path.of("data.txt");
+     * write(path, text); // @highlight substring="write"
+     * var data = read(path);
+     * print(data);
+     * }
+     * will print on the output stream as:
+     * {@snippet lang="text":
+     * "Hope" is the thing with feathers
+     * That perches in the soul
+     * And sings the tune without the words
+     * And never stops - at all
+     *
+     * }
+     *
+     * @implSpec Line terminators are normalized the platform line separator.
+     *
+     * @param path  path of file to be written
+     * @param content  string content of the file
+     * @throws UncheckedIOException wrapping an IOException if an io error occurs.
+     */
+    public static void write(Path path, String content) {
+        Objects.requireNonNull(path, "path must not be null");
+        Objects.requireNonNull(content, "content must not be null");
+        try (PrintWriter out = new PrintWriter(path.toFile())) {
```

Comment:
Could this use Files.writeString(Path, String)?

Reply:
Thx